packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -343,11 +343,11 @@ export const MobileNav: FC = () => {
       <aside
         className={cn(
           classes.aside,
-          '_fixed _top-[--nextra-navbar-height] _w-full _bottom-0 _z-10 _overscroll-contain',
+          '_fixed _inset-y-0 _w-full _pt-[--nextra-navbar-height] _z-10 _overscroll-contain',
```

Comment:
```suggestion
          '_fixed _inset-0 _pt-[--nextra-navbar-height] _z-10 _overscroll-contain',
```
let's use ` _inset-0` so we can remove `_w-full`